app/build.gradle

```diff
@@ -7,6 +7,8 @@ plugins {
 dependencies {
     implementation project(":api")
 
+    implementation "net.corda:corda-http-rpc-client:5+"
```

Comment:
NB: This adds a dependency on the artefact built in `corda-runtime-os`.
@mnesbit - we need to discuss whether this is allowed/acceptable.